lntest/node.go

```diff
@@ -984,6 +985,11 @@ func (hn *HarnessNode) FetchNodeInfo() error {
 		return err
 	}
 
+	// Check that the node is fully started.
+	if !info.Started {
```

Comment:
See prior comment re just having the node creation pause until it's fully started up. 

Also same comment here re just using the exiting status endpoint (which was intended for this type of synchronization).